plugins/account_history/include/golos/plugins/account_history/history_object.hpp

```diff
@@ -22,13 +22,102 @@ namespace plugins {
 namespace account_history {
 
     enum account_object_types {
-        account_history_object_type= (TAG_SPACE_ID << 8),
-        account_history_object_type_isias = (TAG_SPACE_ID << 8) + 1,
+        account_history_object_type = (ACCOUNT_HISTORY_SPACE_ID << 8),
+        operation_object_type = (ACCOUNT_HISTORY_SPACE_ID << 8) + 1,
     };
+
     using namespace golos::chain;
     using namespace chainbase;
-    // using chainbase::object_id;
-    // using account_history_id_type = object_id<account_history_object>;
+
+
+    class operation_object
+            : public object<operation_object_type, operation_object> {
+    public:
+        operation_object() = delete;
+
+        template<typename Constructor, typename Allocator>
+        operation_object(Constructor &&c, allocator <Allocator> a)
+                :serialized_op(a.get_segment_manager()) {
+            c(*this);
+        }
+
+        id_type id;
+
+        transaction_id_type trx_id;
+        uint32_t block = 0;
+        uint32_t trx_in_block = 0;
+        uint16_t op_in_trx = 0;
+        uint64_t virtual_op = 0;
+        time_point_sec timestamp;
+        buffer_type serialized_op;
+    };
+
+    using operation_id_type = object_id<operation_object>;
+
+    struct by_location;
+    struct by_transaction_id;
+    using operation_index = multi_index_container <
+        operation_object,
+            indexed_by<
+                ordered_unique <
+                    tag < by_id >,
+                    member <
+                        operation_object,
+                        operation_id_type,
+                        &operation_object::id
+                        >
```

Comment:
In one line: `member<operation_object, operation_id_type, &operation_object::id>`

Reply:
Fixed